packages/gatsby/src/utils/schemaUtils.js

```diff
@@ -0,0 +1,224 @@
+// @ts-check
+
+import {polyfillDOMImplementation}                                                                      from 'interweave-ssr';
+import {Markup}                                                                                         from 'interweave';
+import jref                                                                                             from 'json-ref-lite';
+import {cloneDeep, merge}                                                                               from 'lodash';
+import marked                                                                                           from 'marked';
+
+import React                                                                                            from 'react';
+
+import {SymlContainer, SymlMain, SymlScalar, SymlScalarProperty, SymlObjectProperty, SymlArrayProperty} from '../components/syml';
+
+
+polyfillDOMImplementation();
+
+
+export const defaultCommentConfiguration = {
+  /** @type {boolean | undefined} */
+  margin: undefined,
+  /** @type {string | undefined} */
+  anchor: undefined,
+  overrides: {
+    /** @type {unknown} */
+    default: undefined,
+  },
+};
+
+/**
+ * @param {string} [markdown]
+ * @returns {JSX.Element | undefined}
+ */
+export const renderHtmlFromMarkdown = (markdown) => (
+  markdown
+    ? (
+      <Markup
+        content={marked(markdown)}
+        tagName={`fragment`}
+      />
```

Comment:
I'm not familiar with `interweave` - why not just use `dangerouslySetInnerHTML`, since we control the input? It seems to be responsible for most of the new deps in this diff.

Reply:
You're right. I'm not too familiar with react, so, when I googled it, I found that Interweave is usually the recommended way, but I forgot that, in this case, we control the input.